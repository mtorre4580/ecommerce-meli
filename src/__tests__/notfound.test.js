import React from 'react';
import NotFound from '../components/commons/notfound/notfound';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';

describe('<NotFound />', () => {

    it('should render snapshot', () => {
        const component = renderer.create(
            <Router>
                <NotFound />
            </Router>
        );
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('renders with title not found', () => {
        const wrapper = shallow(<NotFound />);
        const title = <h2 className='title-err'>Parece que esta p√°gina no existe</h2>;
        expect(wrapper.contains(title)).toEqual(true);
    });
     
 });



