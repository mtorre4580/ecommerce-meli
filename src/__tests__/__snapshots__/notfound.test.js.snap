// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should render snapshot 1`] = `
<div
  className="not-found"
>
  <h2
    className="title-err"
  >
    Parece que esta página no existe
  </h2>
  <a
    aria-current="true"
    className="active"
    href="/"
    onClick={[Function]}
    style={Object {}}
  >
    Ir a la página principal
  </a>
</div>
`;
